golem-cluster.yaml

```diff
@@ -36,29 +36,34 @@ provider:
         outbound_urls: ["https://pypi.dev.golem.network"]
 
         # Minimal values of parameters offered by Golem provider 
-        min_mem_gib: 0
-        min_cpu_threads: 0
-        min_storage_gib: 0
-
-      budget_control:
-        # Estimated expected load and duration for worker that tells budget control to pick the least expensive Golem provider offers first.
-        # If not provided, offers will be picked at random.
-        # Both values need to be defined or undefined together.
-        expected_cpu_load: 0.8
-        expected_duration_minutes: 20
-
-        # Amount of GLMs for expected usage which Golem provider offer will be rejected if exceeded.
-        # Requires "expected_cpu_load" and "expected_duration_minutes" parameters to take effect.
-        max_expected_usage_cost: 1.5
+        min_mem_gib: 1
+        min_cpu_threads: 1
+        min_storage_gib: 1
+
+        # Maximal values of parameters offered by Golem provider
+        max_cpu_threads: null
+
+      # Golem providers differ from each other
+      per_cpu_budget:
+        expected_usage:
+          # Estimated expected load and duration for worker that tells budget control to pick the least expensive Golem provider offers first.
+          # If not provided, offers will be picked at random.
+          # Both values need to be defined, undefined or nulled together.
+          cpu_load: 0.8
+          duration_hours: 0.3 # 20 minutes
```

Comment:
18 minutes ?

Reply:
as can be observed @approxit is confused not only by seconds ;)))